config/manifests/inferencepool-resources.yaml

```diff
@@ -107,6 +107,10 @@ rules:
   - subjectaccessreviews
   verbs:
   - create
+- nonResourceURLs:
+  - "/metrics"
+  verbs:
+  - get
```

Comment:
The [metrics guide](https://gateway-api-inference-extension.sigs.k8s.io/guides/metrics/) states that this rule should be added to a separate ClusterRole. The guide needs to be updated or this should be removed. Since this change is out of scope, I suggest removing this change and creating an issue if you would like to change the metrics RBAC configuration.